user_docs/en/changes.md

```diff
@@ -48,6 +48,7 @@ Localisation data for emojis has been added for Belarusian and Bosnian.
   * Fixed a problem where NVDA sometimes freezes completely when using SAPI5 voices. (#18298, @gexgd0419)
   * Fixed a problem where NVDA fails to start with a SAPI5 Eloquence voice and falls back to OneCore voices. (#18301, @gexgd0419)
   * Fixed audio gaps in speech when using some SAPI5 voices with WASAPI and rate boost enabled. (#17967, @gexgd0419)
+  * When unicode normalization is on, navigating by character will again correctly announce combining diacritic characters like acute (\u0301). (#18722, @LeonarddeR)
```

Comment:
`\u0301` doesn't appear to be formatted correctly in either markdown or HTML - can you please ensure the symbol itself is used as well for a clearer example. e.g.

```suggestion
  * When unicode normalization is on, navigating by character will again correctly announce combining diacritic characters like acute (`◌́ \u0301`). (#18722, @LeonarddeR)
```